.github/workflows/main.yml

```
@@ -1,79 +1,53 @@
 name: Java CI/CD
 
-on:
-  pull_request:
-    branches:
-      - '**'
-  push:
-    branches:
-      - '**'
-    tags:
-      - 'v*'
+on: [ push ]
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)